autorest/azure/environments.go

```diff
@@ -163,7 +170,8 @@ var (
 		ContainerRegistryDNSSuffix:   "azurecr.us",
 		TokenAudience:                "https://management.usgovcloudapi.net/",
 		APIManagementHostNameSuffix:  "azure-api.us",
-		SynapseEndpointSuffix:        NotAvailable,
+		SynapseEndpointSuffix:        ".dev.azuresynapse.usgovcloudapi.net",
```

Comment:
Should this have a leading . ?
```suggestion
		SynapseEndpointSuffix:        "dev.azuresynapse.usgovcloudapi.net",
```

Reply:
Good catch, fixed.